libs/server/Resp/ACLCommands.cs

```diff
@@ -17,6 +16,37 @@ namespace Garnet.server
     /// </summary>
     internal sealed unsafe partial class RespServerSession : ServerSessionBase
     {
+        private bool IsACLAuthenticatorEnabled()
```

Comment:
I would not expect an `IsXXX()` method to _write a response_.  Something like `ValidateXXX` or `EnforceXXX` would communicate that a response might be written.

Reply:
Beyond that first blush of confusion, LGTM.

Reply:
Makes sense. I just pushed a change using ValidateACLAuthenticator and ValidateACLFileUse names